build/configs/esp32_DevKitC/tc/defconfig

```diff
@@ -177,11 +177,11 @@ CONFIG_ESP32CORE_XTAL_40MHZ=y
 CONFIG_DEV_TTY_USB0=y
 # CONFIG_DEV_TTY_USB1 is not set
 CONFIG_ESPTOOLPY_PORT="/dev/ttyUSB0"
-CONFIG_ESPTOOLPY_BAUD_115200B=y
+# CONFIG_ESPTOOLPY_BAUD_115200B is not set
```

Comment:
What is difference between CONFIG_ESPTOOLPY_BAUD_115200B and CONFIG_ESPTOOLPY_BAUD=2000000.
How about using CONFIG_USB*_BAUD=115200 or 2000000.
(* means 1~N)
If I am wrong, Please let me know.

Reply:
In menuconfig, items CONFIG_ESPTOOLPY_BAUD_115200B/.../CONFIG_ESPTOOLPY_BAUD_2000000B could be chosen, after CONFIG_ESPTOOLPY_BAUD_2000000B is chosen, CONFIG_ESPTOOLPY_BAUD is set with value 2000000.
CONFIG_ESPTOOLPY_BAUD is used in build/tools/esp32/esptool_py/Makefile. Now, CONFIG_ESPTOOLPY_BAUD is same with esp-idf. And for different boards esp32-DevKitC and esp-wroover-kit, the BAUD items should be defined to CONFIG_USB0_BAUD or CONFIG_USB1_BAUD, and the makefile may be different. If new board is add, new BAUD items may be added. So, it is not so convenient. And it is better to keep it.